pkgs/applications/graphics/processing/default.nix

```
@@ -1,58 +1,133 @@
-{ stdenv, fetchFromGitHub, fetchurl, xmlstarlet, makeWrapper, ant, jdk, rsync, javaPackages, libXxf86vm, gsettings-desktop-schemas }:
-
-stdenv.mkDerivation rec {
+{ stdenv, lib, jdk, patchelf, makeWrapper, xlibs, zip, unzip, rsync
```

[1/2]
```suggestion
{ stdenv, lib, jdk, patchelf, makeWrapper, xlibs, zip, unzip, rsync, fetchzip
```
[`fetchTarball`](https://nixos.org/nix/manual/#ssec-builtins) is meant to pull Nix expressions at evaluation time. If Nixpkgs `fetchzip` is used instead it should help with the OfBorg error.